dask/bytes/tests/test_local.py

```diff
@@ -131,7 +131,7 @@ def test_read_bytes():
 def test_read_bytes_sample_delimiter():
     with filetexts(files, mode='b'):
         sample, values = read_bytes('.test.accounts.*',
-                                    sample=80, delimiter=b'\n')
+                                    sample='80 B', delimiter=b'\n')
```

Comment:
Maybe want to add a test with different size to ensure that this is working as expected - should get different number of lines.

Reply:
Done?

Reply:
That will do.